src/pages/Material/index.js

```diff
@@ -1,15 +1,63 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
-import { compareDesc } from 'date-fns';
+import React, { useEffect, useState } from 'react';
+import { compareDesc, formatRelative, parseISO, toDate } from 'date-fns';
+import { pt } from 'date-fns/locale';
 import { Main, Materials } from './style';
 import Card from '../../components/Card';
 import ActivityHistory from '../../components/ActivityHistory';
+import api from '../../services/api';
 
 const Material = () => {
-  const materials = useSelector((state) => state.materials);
-  const loans = useSelector((state) =>
-    state.loans.slice().sort((a, b) => compareDesc(a.updated_at, b.updated_at))
-  );
+  const [materials, setMaterials] = useState([]);
+  const [loans, setLoans] = useState([]);
+
+  useEffect(() => {
+    api
+      .get('/materials')
+      .then((response) => {
+        if (response.status === 200) {
+          setMaterials(response.data);
+        }
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, []);
+
+  useEffect(() => {
+    api
+      .get('/loans')
+      .then((response) => {
+        if (response.status === 200) {
```

Comment:
The ```then``` method always return a response with a status code valid. So status code greater than 400 will catch by ```catch``` method.

If you want code that of one simple way, you can do this by using async wait statement.

Example:
```js
useEffect(() => {
  async function fetchLoan() {
    const res = await api.get('/loans')
    // rest of your code
  }
  
  fetchLoan() 
})
```

You also can do the fetching in the same effect method. :point_up: